compiler/rustc_session/src/options.rs

```diff
@@ -1387,6 +1387,8 @@ options! {
         "allow the linker to link its default libraries (default: no)"),
     dlltool: Option<PathBuf> = (None, parse_opt_pathbuf, [UNTRACKED],
         "import library generation tool (ignored except when targeting windows-gnu)"),
+    ehcont_guard: bool = (false, parse_bool, [TRACKED],
+        "generate Windows EHCont Guard tables"),
```

Comment:
This is an insta-stable option, and would need an FCP to be added. Can you instead add it as an unstable option first?

Reply:
Certainly.  Would you prefer this as an unstable codegen option, or an unstable target feature?

Reply:
I'm not sure what's better, I'll let the others decide.

Reply:
The normal path for adding codegen options like this is first as a `-Z` flag and then FCP the `-C` stabilization. 

@sivadeilra if you don't mind changing this to be an unstable option currently, I'm happy to r+ this!

Reply:
Converted to `-Z ehcont-guard`